gaeun/app.js

```diff
@@ -62,11 +61,150 @@ app.post("/user/signup", async (req, res) => {
   res.status(201).json({ message: "userCreated" });
 });
 
+////////////////////////////
+// Assignment3 - 게시글 등록 //
+////////////////////////////
+
+app.post("/posts", async (req, res) => {
+  const { title, postImage, content, userId } = req.body;
+  console.log(title, postImage, content, userId);
+
+  const post = await myDataSource.query(
+    `INSERT INTO posts(
+      title,
+      post_image,
+      content,
+      user_id
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [title, postImage, content, userId]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+////////////////////////////////
+// Assignment4 - 전체 게시글 조회 //
+////////////////////////////////
+
+app.get("/posts", async (req, res) => {
+  await myDataSource.manager.query(
```

Comment:
```suggestion
  await myDataSource.manager.query(
```
`manager`를 사용하신 이유가 있을까요?

Reply:
위스터디에서 제공된 동영상에서 manager를 사용하길래 따라 쳤습니다........

https://orkhan.gitbook.io/typeorm/docs/entity-manager-api
https://www.tutorialspoint.com/typeorm/typeorm_working_with_entity_manager.htm
구글링해보니 이 두 링크 외에는 딱히 Manager에 관해 설명한 것이 없는 것 같습니다. 

제가 이해한 바로는 Typeorm을 사용할 때 manager를 사용함으로써 여러개의 엔티티에 접근할 수 있게 하는 것 같은데... 맞나요?

그리고 지금은 single entity만 이용하기 때문에 manager는 삭제하도록 하겠습니다!